packages/sdk-router/src/constants/addresses.ts

```diff
@@ -40,7 +40,7 @@ export const ROUTER_ADDRESS_MAP: AddressMap = generateAddressMap(
 /**
  * SynapseCCTP contract address for all chains except ones from CCTP_ROUTER_EXCEPTION_MAP.
  */
-const CCTP_ROUTER_ADDRESS = '0xD359bc471554504f683fbd4f6e36848612349DDF'
+const CCTP_ROUTER_ADDRESS = '0xd5a597d6e7ddf373a92C8f477DAAA673b0902F48'
```

Comment:
Let's revert this change: this is dealt with in #1617

Reply:
Updated in https://github.com/synapsecns/sanguine/pull/1622/commits/49dae15391e794f0226ce87930e6bc621881e644